### DISCUSSION TITLE
Sort completions in columns instead of rows

### DISCUSSION THREAD

**Discussion by h0adp0re** [marlonrichert/zsh-autocomplete#267] [Q&A] [answered]
## What do you want?

Sort completions in columns instead of rows

## Why do you want this?

Easier on the eyes.

## Who else would benefit from this?

Probably everyone.

### Given the following situation:
Type in a command with a large list of possible completions.

### When I perform the following steps:
Tab through the completions.

### Then I expect the following to happen:
The completions are iterated in columns instead of rows, it's qute hard to parse visually as of right now.

https://user-images.githubusercontent.com/11251772/118971845-8e53c200-b978-11eb-9858-1c4d77110b04.mov

**Comment by marlonrichert** [comment, marked as answer]
Add to your `.zshrc` file:
```zsh
zstyle ':completion:*:*' list-rows-first no
```
However, please be aware that this worsens `zsh-autocomplete`'s ability to estimate how many completions can fit on screen. If you enable this feature, you might run into the occasional `zsh: do you wish to see all 71 possibilities (73 lines)?` eating some of the keys you type. Caveat emptor.

**Reply by h0adp0re** [reply]
Yup, this solves it. Thanks!
